macos/FluentUITestViewControllers/TestViewControllers.swift

```diff
@@ -11,6 +11,7 @@ public struct TestViewController: Identifiable {
 	public let type: NSViewController.Type
 }
 
+@available(OSX 10.14, *)
```

Comment:
What is this tag for?

Reply:
The Button view controller has a bunch of 10.14+ features.  This is the topmost-level availability tag that allows us to load up the button view controller class.  Somehow setting the deployment target to 10.14+ was not sufficient to let me get rid of all these availability tags.

Reply:
Fixed.  I must have picked up a change to the minimum OS version recently and was able to remove this stuff.